folksonomy/api.py

```diff
@@ -1103,65 +1112,123 @@ async def get_user_info(user: User = Depends(get_current_user)):
         "user": user_roles["user"],
     }
 
+@app.post("/admin/value/check-clash", response_model=PropertyClashCheck, tags=["Admin - Value Management"])
+async def check_value_clash(
```

Comment:
Ok sorry, I induced you in error because I ask for this method, while there can currently only be one value per property… so no clash possible.

So the only think we need is already there in stats API: a count of the values so that we can say: "This operation will rename 234 values "red" for property "color" to "RED" (which have currently 22 occurences)"

234 would be the result of /products/stats?k=color&v=red
22 would be the result of /products/stats?k=color&v=RED

So we don't need this method, and you should revert back to the logic of your first implementation on value rename.